src/main/java/uk/gov/companieshouse/api/strikeoffobjections/controller/ObjectionController.java

```diff
@@ -40,16 +40,17 @@
 import java.util.Map;
 import java.util.stream.Collectors;
 
+import static uk.gov.companieshouse.api.strikeoffobjections.service.impl.ERICHeaderFields.ERIC_REQUEST_ID;
```

Comment:
Moved these out of here into their own class as one of them is also needed by the new interceptor and I had to define a new one too. Made sense to keep them all together I thought.